src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/PInvokeModuleFixupNode.cs

```diff
@@ -37,9 +37,7 @@ public override ObjectData GetData(NodeFactory factory, bool relocsOnly = false)
             ObjectDataBuilder builder = new ObjectDataBuilder(factory);
             builder.DefinedSymbols.Add(this);
 
-            ISymbolNode nameSymbol = factory.Target.IsWindows ?
```

Comment:
I'm indifferent about this change - if you think this was a premature optimization, I'm up for removing it.

This was the only use of the `ConstantUtf16String` method, so we should delete that too.